packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4884,10 +4896,8 @@ export default class Meeting extends StatelessWebexPlugin {
           this.setUpVoiceaListeners();
         }
 
-        if (this.getCurUserType() === 'host') {
-          // @ts-ignore
-          await this.webex.internal.voicea.turnOnCaptions(options?.spokenLanguage);
-        }
+        // @ts-ignore
+        await this.webex.internal.voicea.turnOnCaptions(options?.spokenLanguage);
```

Comment:
Is it possible to add some ut for this.

Reply:
Thanks, @Shreyas281299 - Addressed this comment [in this line of code](https://github.com/webex/webex-js-sdk/pull/3783/files#diff-8dae4d0be73ef303fd51e1243f57b86f234ed724645f1ccdea3b26d0e2e51f57R1019).